.github/workflows/release.yaml

```diff
@@ -56,7 +55,9 @@ jobs:
           draft: true
           prerelease: true
       - name: Upload cloud-hypervisor
-        if: github.event_name == 'create' && github.event.ref_type == 'tag'
+        if: |
```

Comment:
Can you share a reference to explain why would a prefix of `|` be needed here?

Reply:
It is used in workflow `if` to write multi-line if conditions. It improves readability to split conditions to multiple lines instead of having one long line. More info [here](https://github.com/orgs/community/discussions/25641).

According to [YAML spec](https://yaml.org/spec/1.2.2/#81-block-scalar-styles) `|` marks literal block scalar header.